### DISCUSSION TITLE
Możliwość przesyłania dokładnie takich samych kart w edytorze

### DISCUSSION THREAD

**Discussion by itchystone** [emkarcinos/WMIAdventure#406] [Codding]
Może warto by było w przyszłości na backendzie przed przyjęciem karty sprawdzić, czy nie ma już dokładnie takiej samej karty oczekującej do akceptacji?

Przykład:

![image](https://user-images.githubusercontent.com/43492764/126631546-ee267a02-fbd9-4895-a462-e0c445373ed8.png)

W edytorze mogę sobie spamić przycisk wyślij i backend wszystko przyjmie:
![image](https://user-images.githubusercontent.com/43492764/126631678-69fe6b55-b99b-4a58-ad5d-fd85b1992bbc.png)

Warto by się było móc uchronić przed takim zasypem zbędnych propozycji identycznych kart.

**Comment by emkarcinos** [comment]
Moim zdaniem powinna być możliwość przysłania dwóch takich samych kart.
Co jeżeli akurat dwóch studentów wpadnie na pomysł na taką samą kartę? Dlaczego mielibyśmy odrzucać bez żadnej kontroli tego, kto spóźnił się względem drugiego? Wszystkie karty powinny trafiać do akceptacji.

**Reply by itchystone** [reply]
Rozumiem, myślę jednak, że jeśli w przyszłości dodamy nagrody za tworzenie kart to trzeba będzie rozważyć co robić z identycznymi kartami, ale zostawmy to na później.
